akka-typed/src/main/scala/akka/typed/scaladsl/Actor.scala

```diff
@@ -307,23 +300,25 @@ object Actor {
    * for logging or tracing what a certain Actor does.
    */
   final case class Tap[T](
-    signal:   (ActorContext[T], Signal) ⇒ _,
-    mesg:     (ActorContext[T], T) ⇒ _,
-    behavior: Behavior[T]) extends Behavior[T] {
+    onSignal:  (ActorContext[T], Signal) ⇒ _,
+    onMessage: (ActorContext[T], T) ⇒ _,
```

Comment:
I just noticed that we have a different ordering of `onSignal` and `onMessage` in various behavior builders. I parameter related to message should go first, and then the one related to signals.